sdk/trace/batch_span_processor.go

```diff
@@ -121,6 +129,34 @@ func NewBatchSpanProcessor(exporter SpanExporter, options ...BatchSpanProcessorO
 	return bsp
 }
 
+func (bsp *batchSpanProcessor) configureSelfObservability() {
+	mp := otel.GetMeterProvider()
+	if !x.SelfObservability.Enabled() {
+		mp = metric.MeterProvider(noop.NewMeterProvider())
+	}
+	meter := mp.Meter(
+		selfObsScopeName,
+		metric.WithInstrumentationVersion(version()),
+	)
+
+	queueSizeCounter, err := meter.Int64ObservableUpDownCounter("otel.sdk.batch_span_processor.queue_size",
```

Comment:
I just realized that there isn't a good way to distinguish different instances of the batch span processor.  I think we might want to use a scope attribute to do that, but i'm not sure what the attribute should be.  We don't have any unique identifiers for exporters or processors.  We could generate one, but that doesn't seem very helpful.

Reply:
The proposed "otel.sdk.component.name" attribute has a suffix that is meant to identify the instance of the BSP.